src/videojs-contrib-hls.js

```diff
@@ -205,6 +229,12 @@ export default class HlsHandler extends Component {
       this.tech_.trigger('loadedmetadata');
     });
 
+    // the bandwidth of the primary segment loader is our best
+    // estimate of overall bandwidth
+    this.masterPlaylistController_.on('progress', function() {
```

Comment:
I think we're only triggering progress on the tech. That said, MPC should probably trigger it on itself too (or only trigger it on itself, and let this guy trigger it on the tech).